build_sdist/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Build sdist'
+description: 'Build a Python package source distribution.'
+
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v3
+
+    - name: Build SDist
+      shell: bash
+      run: pipx run build --sdist
```

Comment:
Not heard of `pipx` until now. Why use it here?

Reply:
It runs the build in an isolated environment